src/components/BookCard.md

```diff
@@ -0,0 +1,11 @@
+Basic BookCard:
+
+```js
+<BookCard
+    coverUrl="https://books.google.de/books/content?id=0zQQkc8uHXMC&printsec=frontcover&img=1&zoom=5&edge=curl&imgtk=AFLRE71nrLI7QflNydctB5_NxDqjtWAeojCAQTH3-KWEZk_8b1j6mGETIEZSKAjpT7zas_3IhuOh9mGG099CzVQgzK2-eRY9ff0ENS_9vxtbe7u6exBCEvsj-rap3i1mtMytcdqO7SJ-"
```

Comment:
Please download a dummy image and save it (e.g. in `public/styleguide/dummy-bookcover.jpg`)to prevent the need of an external call.